Added the missing any and size_t types

pouyakary opened the pull request:
The types `size_t` and `any` were not defined in the syntax so this patch adds them:

<br>

<img width="644" alt="Screen Shot 1400-05-02 at 18 08 45" src="https://user-images.githubusercontent.com/2157285/126870164-3fa5ef45-5a55-41a3-98ed-aca77f0ea3fd.png">

danieldaeschle commented:
any is not needed imo.
Should be removed from V

danieldaeschle commented on syntaxes/v.tmLanguage.json:
```diff
@@ -267,7 +267,7 @@
 						},
 						{
-							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune)(?=\\s*\\()",
+							"match": "(bool|byte|byteptr|charptr|voidptr|string|ustring|rune|size_t)(?=\\s*\\()",
```
Can you remove ustring please? It was removed recently.

pouyakary replied:
Yeah sure.

danieldaeschle replied:
Thanks :)

pouyakary commented:
I am going to implement the #246  and #247, before doing so I also added the [CSpell](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) and fixed two little spellings.

danieldaeschle commented:
Running cspellchecker is fine but please remove it from the project

danieldaeschle commented:
Please implement the other things in extra PRs that i can merge this

danieldaeschle commented on .vscode/settings.json:
```diff
@@ -0,0 +1,18 @@
+{
```
Remove this file

danieldaeschle commented on .vscode/extensions.json:
```diff
@@ -0,0 +1,5 @@
+{
```
Remove this file

pouyakary commented:
Sure do
